DOCS/man/options.rst

```diff
@@ -1557,6 +1557,11 @@ Video
     software decoding and hardware decoding methods that copy the video back to
     system memory support all values between 0 and 359.
 
+``--video-crop=<w:h:x:y>`` ``--video-crop=<w:h>``
```

Comment:
Oh this is kind of a personal bikeshed, but I'd prefer imagemagick-like syntax here honestly. i.e. `wxh+x+y`

Reply:
Ok, sure make sense. I copied ffmpeg's [vf crop](https://ffmpeg.org/ffmpeg-filters.html#crop) syntax.

Reply:
done